tests/e2e/Provide_Existing_Coverage/.gitignore

```diff
@@ -0,0 +1,2 @@
+infection-coverage/junit.xml
+infection-coverage/coverage-xml/
```

Comment:
We have to generate these on the fly: there were some relative-absolute path related issues, which are best to be avoided completely. Generating these takes no time at all.